### PR TITLE
Issue #186: Business routes are incorrectly tagged

### PR DESCRIPTION
**Pull request by tebrown**
This PR is to address issue #186.  'Business' routes (usually tagged as 'Business <HWY>' number are incorrectly tagged.  In the initial ticket, I indicated the 'Business 190' should be the street name, but I believe the streetName should be '190' and the 'StreetNamePreType' should be 'Business'.  I trained this with 5 real addresses in Covington, LA, and validated it was 4 addresses of a similar format in other regions.  All unit tests passed.

### CODE REVIEW

**Comment by jeancochrane** [comment]
Thanks for this PR @tebrown! I'll take a look today.

File: measure_performance/test_data/new_tests.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<AddressCollection>
```
Comment (jeancochrane) [inline measure_performance/test_data/new_tests.xml]: We collect test data in in `measure_performance/test_data/labeled.xml`, the "master" testing file. If you could append these addresses to the end of that file it'd help keep the directory clean.

File: training/new_addresses.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<AddressCollection>
```
Comment (jeancochrane) [inline training/new_addresses.xml]: The master training file is `training/labeled.xml`, so these addresses should get appended there.

File: training/new_addresses.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  <AddressString><AddressNumber>1200</AddressNumber> <StreetNamePreType>business</StreetNamePreType> <StreetName>190</StreetName> <OccupancyType>ste</OccupancyType> <OccupancyIdentifier>42,</OccupancyIdentifier> <PlaceName>COVINGTON,</PlaceName> <StateName>LA</StateName> <ZipCode>70433</ZipCode></AddressString>
+  <AddressString><AddressNumber>1200</AddressNumber> <StreetNamePreType>business</StreetNamePreType> <StreetName>190</StreetName> <OccupancyType>ste</OccupancyType> <OccupancyIdentifier>41,</OccupancyIdentifier> <PlaceName>COVINGTON,</PlaceName> <StateName>LA</StateName> <ZipCode>70433</ZipCode></AddressString>
+  <AddressString><AddressNumber>1200</AddressNumber> <StreetNamePreDirectional>business</StreetNamePreDirectional> <StreetNamePreType>190</StreetNamePreType> <OccupancyType>ste</OccupancyType> <OccupancyIdentifier>42,</OccupancyIdentifier> <PlaceName>COVINGTON,</PlaceName> <StateName>LA</StateName> <ZipCode>70433</ZipCode></AddressString>
```
Comment (jeancochrane) [inline training/new_addresses.xml]: In the first five training examples, `business` is tagged as `StreetNamePreType` and `190` is tagged as `StreetName`, and I agree that this seems like the proper pattern for this type of address. But here `business` is tagged as `StreetNamePreDirectional` and `190` is `StreetNamePreType`... any reason for the discrepancy?

**Comment by jeancochrane** [comment]
Left a few comments, but otherwise this looks great! Let me know if you can make these changes.

I learned something new about naming standards for U.S. Highways, so you know it's been a good day.

File: training/new_addresses.xml (hunk 3 above)
Comment (tebrown) [inline training/new_addresses.xml]: I think this was an error when I was learning how do do this. I'll fix it up!

File: training/new_addresses.xml (hunk 2 above)
Comment (tebrown) [inline training/new_addresses.xml]: roger. Will do.

**Comment by tebrown** [comment]
US Address schemes are a nightmare, and this project has saved me countless hours in doing address lookups from GIS databases.  Thank you for your work!

**Comment by tebrown** [comment]
Thanks!  Cheers!

**Comment by jeancochrane** [comment]
Merged. Glad usaddress was useful to you @tebrown! Thanks for helping out :)

**Comment by tebrown** [comment]
@jeancochrane - what is the process for closing ticket?  Do you do it when you tag a new release, or is it done by the submitter upon merging?

**Comment by jeancochrane** [comment]
Thanks for the reminder! We don't have any specific policy, but I like to keep the issues pruned so I usually wind up doing it. Next time around, you should feel free to close once your PR gets accepted.

**Pull request merged**